shell/imports/server/migrations.js

```diff
@@ -787,6 +787,49 @@ function setIpBlacklist(db, backend) {
   }
 }
 
+function notifyIdentityChanges(db, backend) {
+  // Notify users who might be affected by the identity model changes.
+  //
+  // Two types of users are affected:
+  // - Users who have multiple identities with differing names.
+  // - Users who have identities that are shared with other users.
+  //
+  // However, the second group seems like it must be a subset of the first group. So we only check
+  // for the first.
```

Comment:
Why must the second group be a subset of the first? Suppose I have an account where github:dwrensha is a login identity and email:david@sandstorm.io is a non-login identity, and I have another account where google:dwrenshaw is a login identity and email:david@sandstorm.io is again a non-login identity. There's nothing stopping all of the associated profiles from having the same name "David Renshaw".